doc/user_guide/statements/select.md

```diff
@@ -0,0 +1,163 @@
+    .function(ExasolScalarFunction.RANDOM, "RANDOM_1") //
+    .function(ExasolScalarFunction.RANDOM, "RANDOM_2", ExpressionTerm.integerLiteral(5), ExpressionTerm.integerLiteral(20));
+```
+
+- `arithmetic expression` is a binary arithmetic expression with one of the following arithmetic operator: +, -, *, /.
+Add an arithmetic expression using an `arithmeticExpression( ... )` method.
+You can also set a name for a derived field that contains an arithmetic expression. 
+
+```java
+final Select select = StatementFactory.getInstance().select() //
+    .arithmeticExpression(ExpressionTerm.plus(ExpressionTerm.integerLiteral(1000), ExpressionTerm.integerLiteral(234)), "ADD");
+```
+
+#### `FROM` clause
+
+A `SELECT` statement can contain one `FROM` clause.
+To start a `FROM` clause, use a method `from()` of the `Select` class. `
+FROM` clause appends one or more of following structures:
+
+- `table` is a name of the table. You can append it with `table( ... )` method.
+
+- `table with AS clause`is a name of table with an alias. You can append it with `tableAs( ... )` method.
+
+- `value table` is a mocked table which you define inside the `SELECT` statement. To use this structure you need to create a `ValueTable` first.
+Then you can use `valueTable( ... )` method of the `Select`.
+
+```java
+final Select selectFromTable = StatementFactory.getInstance().select().all();
+select.from().table("table1");
+
+final Select selectFromTableAs = StatementFactory.getInstance().select().all();
+select.from().tableAs("table", "t");
+
+final ValueTable values = new ValueTable(this.select);
+values.appendRow("r1c1", "r1c2").appendRow("r2c1", "r2c2");
+final Select selectFromValueTable = StatementFactory.getInstance().select().all();
+select.from().valueTable(values);
+```
+
+The `FROM` clause also supports different types of `JOIN`:
+
+- join
+- inner join
+- left join
+- right join
+- full join
+- left outer join
+- right outer join
+- full outer join
+
+To add a `JOIN` clause you need to add a left table and then use one of the join methods. For example, `innerJoin( ... )`; 
+
+```java
+final Select selectFromTable = StatementFactory.getInstance().select().all();
+select.from().table("left_table").innerJoin("right_table", "left_table.foo_id = right_table.foo_id");  
+```
+#### `WHERE` clause
+
+A `SELECT` statement can contain one `WHERE` clause with a boolean expression.
+To add a `WHERE` clause, use a method `where( ... )` of the `Select` class. 
+
+```java
+Select select = StatementFactory.getInstance().select();
+select.all().from().table("person");
+select.where(eq(ExpressionTerm.stringLiteral("foo"), ExpressionTerm.stringLiteral("bar")));
+```
+
+#### `LIMIT` clause
+
+A `SELECT` statement can contain one `LIMIT` clause with offset and count.
+To add a `LIMIT` clause, use a method `limit( ... )` of the `Select` class. 
+
+```java
+Select select = StatementFactory.getInstance().select();
+select.all().from().table("t");
+select.limit(1);
```

Comment:
Won't work. We can use limit() with select fluently before from(), it would work but I don't like that it breaks a normal SQL statement sequence, so I won't add it to the guide